package.json

```diff
@@ -20,10 +20,9 @@
     "lodash.camelcase": "^4.3.0",
     "object-assign": "^4.0.1",
     "postcss": "^5.0.6",
-    "postcss-modules-extract-imports": "^1.0.0",
-    "postcss-modules-local-by-default": "^1.0.1",
-    "postcss-modules-scope": "^1.0.0",
-    "postcss-modules-values": "^1.1.0",
+    "postcss-icss-composes": "^2.0.3",
```

Comment:
`postcss-icss-keyframes` is missing, that was splitted into it's own plugin :) for `@keyframes` support 

Maybe `postcss-icss-import` (`@import`) && `postcss-icss-url` (`url()`) aswell